responsibleai/responsibleai/_managers/counterfactual_manager.py

```diff
@@ -145,7 +152,7 @@ def add(self,
             desired_class=None,
             desired_range=None,
             permitted_range=None,
-            features_to_vary=None,
+            features_to_vary='all',
```

Comment:
Is `all` equivalent to `None` in diceml?

Reply:
No I got it wrong in first place. The default in dice-ml was 'all''.

https://github.com/interpretml/DiCE/blob/004dbd508ea2b801b5f3ca26a87a25eafc19e1f0/dice_ml/explainer_interfaces/explainer_base.py#L50